### PR TITLE
Add ictnj to kubeedge adopters

### PR DESCRIPTION
**Pull request by hdf15804299051**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ KubeEdge is used by many companies. If you are using KubeEdge in your organizati
 <a href="http://xh-iot.net/" border="0" target="_blank"><img alt="XH-iot" src="docs/adopters/xh-iot.png" height="70"></a> &nbsp; &nbsp; &nbsp;
 <a href="http://www.jylink.com/" border="0" target="_blank"><img alt="jylink" src="docs/adopters/jylink.png" height="70"></a> &nbsp; &nbsp; &nbsp;<br/><br/>
 
+<a href="http://nwzimg.wezhan.cn/" border="0" target="_blank"><img alt="ictnj" src="docs/adopters/ictnj.png " height="70"></a><br/>
+
+<br/>
```
Comment (fisherxu) [inline ADOPTERS.md:14]:
```suggestion
<a href="http://nwzimg.wezhan.cn/" border="0" target="_blank"><img alt="ictnj" src="docs/adopters/ictnj.png" height="70"></a><br/><br/>
```

Comment (fisherxu) [reply ADOPTERS.md:14]: Is `http://nwzimg.wezhan.cn` an official website for the company?

**Review by fisherxu — COMMENTED** [review]
Please also squash the commits and signoff the commit.

**Comment by fisherxu** [comment]
Use `git rebase HEAD~5 --signoff` to signoff the commits.

File: ADOPTERS.md (hunk 1 above)
Comment (hdf15804299051) [reply ADOPTERS.md]: dco那一步检查失败的原因是什么呢，需要签名吗

**Pull request closed without merging**